issue_tracker_client/src/issue_tracker.py

```diff
@@ -0,0 +1,101 @@
+#Issue Tracker implementation
+
+import requests
```

Comment:
Add docstrings to functions, and explain what the goal of this implementation is.